src/asim/configs/crossborder/trip_mode_choice.csv

```diff
@@ -59,7 +59,7 @@ util_TAXI_cost,Taxi - Fare,@c_cost * ((taxi_base_fare + (df.s2_dist_skims * taxi
 util_TNC_SINGLE_IVT,TNC Single - In-vehicle time,c_ivt * s2_time_skims,,,,,,,,,1,
 util_TNC_SINGLE_wait,TNC Single - Wait time,c_ivt * 1.5 * tnc_single_wait_time,,,,,,,,,1,
 util_TNC_SINGLE_cost,TNC Single - Cost,@c_cost * ((tnc_single_base_fare + (df.s2_dist_skims * tnc_single_cost_per_mile) + (df.s2_time_skims * tnc_single_cost_per_minute).clip(lower=tnc_single_cost_minimum)) * 100 + df.s2_cost_skims),,,,,,,,,1,
-util_TNC Shared_switch,TNC Shared - switch turn-off (depends on data availability),False,,,,,,,,,,-999
+util_TNC Shared_switch,TNC Shared - switch turn-off (depends on data availability),@((~df.nev_available) & (~df.microtransit_available) & (scenarioYear==2022)),,,,,,,,,,-999
```

Comment:
should it be 

@((~df.nev_available) | (~df.microtransit_available)) & (scenarioYear==2022)?

so if only nev is available, we still allow nev?

Reply:
Thanks for checking this. After spending some time looking at it (including making a truth table in Excel), I think it should be &. The condition should only be True if both microtransit and NEV are unavailable, as if it is True then TNC_Shared cannot be selected. If | is used, then TNC_Shared would only be available if both NEV and Microtransit are available, and none of the service areas overlapped in 2022, so using | would result in TNC_Shared not being available anywhere.

|df.nev_available|df.microtransit_available|~df.nev_available|~df.microtransit_available|~df.nev_available & ~df.microtransit_available|~df.nev_available or ~df.microtransit_available|
| ---------- | ---------- | ---------- | ---------- | ---------- | ---------- |
| T | T | F | F | F | F |
| T | F | F | T | F | T |
| F | T | T | F | F | T |
| F | F | T | T | T | T |